contracts/lease/src/lease/liquidation.rs

```diff
@@ -1,10 +1,9 @@
-use serde::{Deserialize, Serialize};
+use serde::Serialize;
 
 use finance::{
     coin::Coin,
     currency::Currency,
-    liability::{Liability, Zone},
-    percent::Percent,
+    liability::{check_liability, Status},
```

Comment:
the calling convention for free functions is to refer to them through the module they are defined in